src/feature/hs/hs_config.c

```diff
@@ -471,25 +475,23 @@ config_generic_service(const config_line_t *line_,
   int have_dir_group_read = 0, have_max_streams = 0;
   int have_max_streams_close = 0;
 
-  tor_assert(line_);
+  tor_assert(line);
   tor_assert(options);
   tor_assert(service);
 
   /* Makes thing easier. */
   config = &service->config;
 
+  if (BUG(strcasecmp(line->key, "HiddenServiceDir")))
```

Comment:
`SECTION_HEADER` ?

Reply:
I think this line is gone in the final version? There are no more "BUG("s in hs_config.c.